web-ui/src/main/resources/catalog/views/default/less/gn_view.less

```diff
@@ -2,6 +2,20 @@
    which has to be applied for all apps
    (ie. admin, search, login, editor). */
 
+// margin
+.gn-margin-top {
```

Comment:
Are these used somewhere?

Reply:
The `.gn-margin-top` and `.gn-margin-bottom` are used. The `.gn-margin-right` and `.gn-margin-left` are not used.